### ISSUE TITLE
Database changes do not arrive in another window

### ISSUE THREAD

**Issue by kabalage** [nolanlawson/worker-pouch#21]
Example:
https://jsbin.com/gefurupogi/1/edit?html,js,console,output

1. Open the example twice
2. Click buttons
3. Watch for changes in the console

Without worker-pouch the change is received in both windows.
With worker-pouch the change is received in only the window where the db.post() call happens.

**Comment by nolanlawson** [comment]
Yep, this bug occurs because there's no way to communicate across the worker boundary. Luckily there are efforts to get change observers into IndexedDB, which would solve this problem.

**Comment by kabalage** [comment]
Thank you, I will look into the workarounds then.
